FOREIGN-KEY-PROTECTION-STATUS.md

```
@@ -0,0 +1,342 @@
+- Controller: `InvoicesController::delete()` (already implemented)
+- Tests: 11 feature tests
+- Commit: b488b25
+
+#### 5. TaxRate (Modules/Products)
+**Foreign Keys:** Products, Invoice Items, Invoice Tax Rates, Quote Items, Quote Tax Rates
+
+**Cannot delete if:**
+- Used in any products
+- Used in any invoice items
+- Used in any invoice tax rates
+- Used in any quote items
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)